scipy/integrate/_ode.py

```diff
@@ -662,19 +662,19 @@ def run(self, f, jac, y0, t0, t1, f_params, jac_params):
         defines the stoppage coordinate of the result.
         """
         raise NotImplementedError('all integrators must define '
-                'run(f, jac, t0, t1, y0, f_params, jac_params)')
+                                  'run(f, jac, t0, t1, y0, f_params, jac_params)')
 
     def step(self, f, jac, y0, t0, t1, f_params, jac_params):
         """Make one integration step and return (y1,t1)."""
         raise NotImplementedError('%s does not support step() method' %
-                self.__class__.__name__)
+                                  self.__class__.__name__)
 
     def run_relax(self, f, jac, y0, t0, t1, f_params, jac_params):
         """Integrate from t=t0 to t>=t1 and return (y1,t)."""
         raise NotImplementedError('%s does not support run_relax() method' %
-                self.__class__.__name__)
+                                  self.__class__.__name__)
 
-    #XXX: __str__ method for getting visual state of the integrator
+        # XXX: __str__ method for getting visual state of the integrator
```

Comment:
This should be indented to the same level as the `def` statement, not the method body.